apps/nextjs/src/pages/index.tsx

```diff
@@ -1,80 +1,104 @@
 import Head from "next/head";
-import { trpc } from "../utils/trpc";
-import type { inferProcedureOutput } from "@trpc/server";
-import type { AppRouter } from "@pickle-app/api";
-import { useAuth, useClerk } from "@clerk/nextjs";
+import { useAuth } from "@clerk/nextjs";
+import { useRouter } from "next/router";
 import Link from "next/link";
-import { useUser } from "@clerk/nextjs";
+import { useState, useEffect } from "react";
 
 const Home: NextPage = () => {
+  const { isSignedIn } = useAuth();
+  const router = useRouter();
+
+  const [isLoading, setIsLoading] = useState(true);
+  //Need to detect if user is signed in and redirect based on that
+  useEffect(() => {
+    if (isSignedIn) {
+      router.push("/app");
+    } else {
+      setIsLoading(false);
+    }
+  }, [isSignedIn, router]);
+
+  const [isDesktop, setIsDesktop] = useState(false);
+  useEffect(() => {
+    const handleResize = () => {
+      setIsDesktop(window.innerWidth > 768); // Desktop is being definer here as viewport size of 769 and above
+    };
+
+    handleResize(); // Check initial window size
+    window.addEventListener("resize", handleResize);
+
+    return () => {
+      //Cleanup after coponent mounte - remove the ebent listener
+      window.removeEventListener("resize", handleResize);
+    };
+  }, []);
+
   return (
     <>
       <Head>
         <title>Welcome</title>
-        <meta name="description" content="Generated by create-t3-app" />
-        <link rel="icon" href="/favicon.ico" />
+        <meta name="description" content="Welcome to The Palisades" />
+        <link rel="icon" href="/pickle-icon.png" />
       </Head>
-      <main className="flex h-screen flex-col items-center bg-gradient-to-b from-[#2e026d] to-[#15162c] text-white">
-        <div className="container flex flex-col items-center justify-center gap-12 px-4 py-8">
-          <h1 className="text-5xl font-extrabold tracking-tight sm:text-[5rem]">
-            Welcome to{" "}
-            <span className="text-[hsl(280,100%,70%)]">Palisade</span>
-          </h1>
-          <AuthShowcase />
+      <main className="flex-grow flex-col items-center overflow-hidden bg-gradient-to-b from-[#6220c0] to-[#15162c] text-white">
+        <div className="flex h-full flex-col items-center justify-between gap-12 px-8 py-16">
+          <div className="w-full">
+            <h1 className="text-5xl font-extrabold tracking-tight sm:text-7xl">
+              Welcome to <br />
+              <span className="text-7xl sm:text-9xl">The</span>{" "}
+              <span className="text-7xl text-[hsl(280,100%,70%)] sm:text-9xl">
+                Palisades
+              </span>
+            </h1>
+          </div>
+          {!!isLoading && (
+            <>
+              <div className="flex h-screen w-screen items-center justify-center">
+                Loading...
+              </div>
+              <div></div>
+            </>
+          )}
+          {!isLoading && !!isDesktop && (
```

Comment:
I'm not sure we "need" Separate desktop/mobile views like this. The benefit of having tailwind is so you can define  different CSS values depending on the size of the screen. For instance using "xl:w-20" would make the width of a component 5rem only if the size of the screen is above 1280px. 

If you wanted completely different looking views for mobile and desktop, the way you have it now would probably be okay. And obviously, if you believe that the way you have it now is better, please let me know so I can understand.